### PR TITLE
Add Inazuma Eleven Go Light/Shadow

### PR DESCRIPTION
**Pull request by ferkrauser**
Adds Inazuma Eleven Go Light/Shadow.

*Automatically generated by PR builder*

### CODE REVIEW

File: inazuma-eleven-go-light-shadow/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
+title = "Inazuma Eleven Go Light/Shadow"
+description = "This game was made by leven 5 and marketed  in 2011 in japan and in 2014 in europe. The history is about Arion Sherwind that is a boy that loves football. He joins the raimon high school at the beggining of the history. He joins to the football club and he take part in the team. So, the football is controlled by Sector fifth and they send a team called the dark nights to destroy the football club. With the help of Mark Evans they must win the imperial road.\nGenre: Football, rol, adventures"
```
Comment (Leo626) [inline inazuma-eleven-go-light-shadow/game.dat]: I'm not a fan of the description. It's long and has grammatical errors. Use the description from the [wikipedia entry](https://en.wikipedia.org/wiki/Inazuma_Eleven_GO_(video_game)). A short 1-2 lines would do.

**Comment by Leo626** [comment]
The boxart is missing the 3DS logo and the screenshot needs to be dual screen. Also the wiki entry you created needs to be in English.

File: inazuma-eleven-go-light-shadow/game.dat (hunk 1 above)
Comment (Schplee) [inline inazuma-eleven-go-light-shadow/game.dat]: Also, please do not include the release dates in the description, as that is redundant.

File: inazuma-eleven-go-light-shadow/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title = "Inazuma Eleven Go Light/Shadow"
```
Comment (Schplee) [inline inazuma-eleven-go-light-shadow/game.dat]: Please change the title to "Inazuma Eleven Go: Light and Shadow" to match the naming scheme of other dual release games.

Please refer to the [Contributions Guide](https://github.com/citra-emu/citra-games-wiki/blob/master/CONTRIBUTING.md) when fixing this issue, as well as those presented by @Leo626.

File: inazuma-eleven-go-light-shadow/game.dat [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+region = "eur"
+release_date = "2014-06-13"
+
+[[testcases]]
+title = "0004000000054300"
+compatibility = "3"
+date = "2017-07-24"
+version = "HEAD-081304e"
+cpu = "Intel Core i5-4440"
+gpu = "AMD Radeon R7 200 series "
```
Comment (TheKoopaKingdom) [inline inazuma-eleven-go-light-shadow/game.dat]: The extra whitespace within the string here isn't needed.

**Comment by chris062689** [comment]
@ferkrauser Has there been an update to this? Thanks.

**Comment by Schplee** [comment]
PR author looks to have abandoned it. Closing.

**Pull request closed without merging**